.changeset/metal-pumas-walk.md

```diff
@@ -0,0 +1,43 @@
+---
+'@astrojs/node': major
+---
+
+# Standalone mode for the Node.js adapter
+
+New in `@astrojs/node` is support for __standalone mode__. With standalone mode you can start your production server without needing to write any server JavaScript yourself. The server starts simply by running the script like so:
+
+```shell
+node ./dist/server/entry.mjs
+```
+
+To enable standalone mode set the new `mode` to `'standalone'` option in your Astro config:
```

Comment:
```suggestion
To enable standalone mode, set the new `mode` to `'standalone'` option in your Astro config:
```
(I ignored the "with standalone mode" above because it still reads fine without a comma. But I think this one really needs one or it reads awkwardly!)